### DISCUSSION TITLE
Different filter numbers in Firefox and Chrome

### DISCUSSION THREAD

**Discussion by homu9** [uBlockOrigin/uBlock-issues#3156] [General] [answered]
Is the filter list different in different browsers?  I have clicked the update icon multi times.
**Firefox**
> uBlock Origin: 1.56.0
> Firefox: 123
> filterset (summary):
>  network: 139288
>  cosmetic: 56097
>  scriptlet: 19006
>  html: 1737
> listset (total-discarded, last-updated):
>  added:
>   CHN-0: 25450-132, 2h.40m
>  default:
>   user-filters: 9-0, never
>   ublock-filters: 37358-157, now
>   ublock-badware: 7750-14, now
>   ublock-privacy: 739-0, now
>   ublock-unbreak: 2293-0, now
>   easylist: 81997-645, now
>   easyprivacy: 50325-849, now Δ
>   plowe-0: 3779-1, now
>   urlhaus-1: 8245-5, now
>   ublock-quick-fixes: 140-1, now
> filterset (user): [array of 9 redacted]
> trustedset:
>  added: [array of 165 redacted]
>  removed:
>   about-scheme
>   edge-scheme
>   moz-extension-scheme
> switchRuleset:
>  added: [array of 2 redacted]
>  removed:
>   no-large-media: behind-the-scene false
>   no-csp-reports: * true
> userSettings:
>  suspendUntilListsAreLoaded: false
>  webrtcIPAddressHidden: true
> hiddenSettings: [none]
> supportStats:
>  allReadyAfter: 2369 ms
>  maxAssetCacheWait: 737 ms
>  cacheBackend: indexedDB
>  unsuspendAfter: 1810 ms

**Chrome**
> uBlock Origin: 1.56.0
> Chromium: 122
> filterset (summary):
>  network: 124705
>  cosmetic: 45619
>  scriptlet: 20187
>  html: 0
> listset (total-discarded, last-updated):
>  removed:
>   CHN-0: null
>  default:
>   user-filters: 0-0, never
>   easylist: 81997-13, now
>   easyprivacy: 50325-6, now
>   plowe-0: 3779-1186, now
>   ublock-badware: 7746-142, now Δ
>   ublock-filters: 37185-355, now
>   ublock-privacy: 743-6, now
>   ublock-quick-fixes: 143-7, now Δ
>   ublock-unbreak: 2236-33, now
>   urlhaus-1: 8255-0, now
> filterset (user): [empty]
> trustedset:
>  added: [array of 105 redacted]
>  removed:
>   chrome-extension-scheme
>   chrome-scheme
>   edge-scheme
> userSettings: [none]
> hiddenSettings: [none]
> supportStats:
>  allReadyAfter: 653 ms (selfie)
>  maxAssetCacheWait: 135 ms
>  cacheBackend: browser.storage.local

**Comment by stephenhawk8054** [comment]
There are filters suitable for Firefox but not Chromium and vice versa, depending on the sites.

**Reply by homu9** [reply]
Thanks for explaination!

**Comment by gorhill** [comment, marked as answer]
You have a different uBO configuration in each browser. For example, you added _"AdGuard Chinese"_ in Firefox, while you disabled _"AdGuard Chinese"_ list in Chrome, and you have a different set of custom filters.

Even so, there is no guaranteed the block count will always be the same with an exact same configuration, and even with the same browser: there is no guarantee a site always load the exact same resources from one moment to the next.

---

> `suspendUntilListsAreLoaded: false`

I suggest you re-enable this. There is a bad advice being spread out there regarding this setting, see <https://github.com/gorhill/uBlock/wiki/Reference-answers-to-various-topics-seen-in-the-wild#should-i-disable-suspend-network-activity-until-all-filter-lists-are-loaded-in-ubo-for-firefoxfirefox-for-android>.

**Reply by homu9** [reply]
Thanks for explaination and suggestion!
